amundsen_application/api/utils/request_utils.py

```diff
@@ -7,3 +8,30 @@ def get_query_param(args: Dict, param: str, error_msg: str = None) -> str:
         msg = 'A {0} parameter must be provided'.format(param) if error_msg is not None else error_msg
         raise Exception(msg)
     return value
+
+
+# TODO: Define an interface for envoy_client
+def make_request_wrapper(method: str, url: str, client, headers, timeout: int):  # type: ignore
```

Comment:
Could you add docstring mentioning that it's expecting envoy client and headers?
Also, how about `timeout` to `timeout_sec` (assuming it's second)?